go.mod

```diff
@@ -16,22 +16,33 @@ require (
 
 require (
 	github.com/go-ole/go-ole v1.2.6 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/klauspost/cpuid/v2 v2.2.5 // indirect
-	github.com/lufia/plan9stats v0.0.0-20230110061619-bbe2e5e100de // indirect
+	github.com/matttproud/golang_protobuf_extensions v1.0.4 // indirect
 	github.com/minio/md5-simd v1.1.2 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/philhofer/fwd v1.1.2 // indirect
-	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
-	github.com/power-devops/perfstat v0.0.0-20221212215047-62379fc7944b // indirect
+	github.com/prometheus/common v0.44.0 // indirect
 	github.com/rs/xid v1.5.0 // indirect
+	github.com/stretchr/testify v1.8.2 // indirect
 	github.com/tklauser/go-sysconf v0.3.11 // indirect
 	github.com/tklauser/numcpus v0.6.0 // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
 	golang.org/x/sync v0.1.0 // indirect
 	golang.org/x/sys v0.10.0 // indirect
+	google.golang.org/protobuf v1.30.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
 
+require (
```

Comment:
Go gave me this warning and I ran `go mod tidy -go=1.16 && go mod tidy -go=1.17`:
```
// github.com/minio/madmin-go/v3 imports
// 	github.com/shirou/gopsutil/v3/cpu tested by
// 	github.com/shirou/gopsutil/v3/cpu.test imports
// 	github.com/stretchr/testify/assert loaded from github.com/stretchr/testify@v1.8.1,
// 	but go 1.16 would select v1.8.2
// github.com/minio/madmin-go/v3 imports
// 	github.com/minio/minio-go/v7/pkg/credentials imports
// 	github.com/json-iterator/go tested by
// 	github.com/json-iterator/go.test imports
// 	github.com/stretchr/testify/require loaded from github.com/stretchr/testify@v1.8.1,
// 	but go 1.16 would select v1.8.2

// To upgrade to the versions selected by go 1.16:
// 	go mod tidy -go=1.16 && go mod tidy -go=1.17
// If reproducibility with go 1.16 is not needed:
// 	go mod tidy -compat=1.17
// For other options, see:
// 	https://golang.org/doc/modules/pruning
```
Please let me know if we should do it differently

Reply:
You need to run go mod tidy -compat=1.20 - we do not support go1.16